test/pkg_finish_test.rb

```diff
@@ -77,19 +80,24 @@
     end
 
     context "given some local repository" do
-      let(:repositories) { [local_repo, remote_repo] }
+      let(:repositories) { [local_repo, local_dvd_repo, remote_repo] }
       let(:base_products) { [sles_product, sled_product] }
 
       let(:local_repo) do
         Y2Packager::Repository.new(repo_id: 1, name: "SLE-12-SP2-0", enabled: true,
-          url: URI("cd://dev/sr0"), autorefresh: false)
+          url: URI("dir:///path/to/extracted_ISO"), autorefresh: false)
```

Comment:
The `dir://` schema makes only sense in installed system, you cannot use it for installing a system (`/` is in inst-sys in RAM disk).

But you can use the `hd`or `iso` schema for installation, like `hd:/dir?device=/dev/sda1`. For details see "Supported URI formats" section in `man zypper`.

Reply:
well, I picked the first one which came to my mind and was not cd/dvd ... but you're right, I'll do it more "real world" example